src/retinaConverter.py

```diff
@@ -0,0 +1,24 @@
+x = []
```

Comment:
Please, explain in the commit message your changes and which new functionality/feature you are adding. Add a description also for this file. And if you really think that this script is needed, ok, but it shouldn't be in `/src` since its not really related to the library, is it?